flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/AggregateITCase.scala

```diff
@@ -259,6 +259,30 @@ class AggregateITCase(mode: StateBackendMode) extends StreamingWithStateTestBase
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
 
+  @Test
+  def testListAggWithExpression():Unit = {
+    val data = new mutable.MutableList[(Int, String)]
+    data.+=((1, "A"))
+    data.+=((1, "B"))
+    data.+=((2, "B"))
+    data.+=((2, "C"))
+    data.+=((2, "C"))
+
+    val t = failingDataSource(data)
+      .toTable(tEnv, 'a, 'b)
+      .groupBy('a)
+      .select('a, 'b.listAgg("-"), call("list_Agg", col("b"), "~"))
```

Comment:
It's better to have SQL functions in uppercase like it's done for others
why does it have underscore in the middle? In SQL Standard it does not have underscore as well as in jira issue